static/src/js/components/DraftList/DraftList.jsx

```diff
@@ -55,11 +55,28 @@ const DraftList = ({ draftType }) => {
 
   // Building a Table using Data in items
   const data = (items.map((item) => {
-    const {
-      conceptId, revisionDate, previewMetadata: {
-        name, longName, shortName, title
-      }
-    } = item
+    let conceptId
+    let revisionDate
+    let title
+    let subTitle
+
+    if (item.ummMetadata) {
```

Comment:
I don't think this check is needed. I feel like we can use just the ummMetadata for all cases.

Reply:
I simplified the logic. Do you think it is better to keep it as is, or change all the graphql queries for get[ConceptType]Drafts?